packages/core-bridge/src/helpers/handles.rs

```diff
@@ -9,6 +9,7 @@ use parking_lot::Mutex;
 
 use super::{BridgeError, BridgeResult, IntoThrow, TryFromJs, TryIntoJs};
 
+#[allow(clippy::doc_overindented_list_items)]
```

Comment:
Checked the formatting of `cargo doc` and everything looked fine so silenced the lint as it looks good in source.